lxd/daemon.go

```diff
@@ -539,6 +603,14 @@ func (d *Daemon) Stop() error {
 		socket.Close()
 	}
 
+	if n, err := d.numRunningContainers(); err != nil || n == 0 {
```

Comment:
I think this is racy, but using `unshare()` will fix it, so I think we can just leave it for now.